ext_emconf.php

```diff
@@ -28,6 +28,7 @@
         'depends' => [
             'typo3' => '8.7.0-8.7.99',
             'php' => '7.0.0-7.2.99',
+            'direct_mail' => '5.2.2-5.9.99'
```

Comment:
In my POV direct_mail should not be a hard dependency of fe_manager.
If you need to make sure, that fe_manager is loaded later, after directmail (in Packagestates) it's enough to put direct mail into suggests array.

Reply:
Yes, i would like it also optional. I have grabbed it from our fe manager extensions, so we have no code to have it optional in there.

Reply:
Hello work this workoround also with direct_mail | 6.0.0-dev and femanager | 5.2.0

Reply:
i mean this direct_mail https://github.com/jokumer/direct_mail/tree/TYPO3_9